tests/chain_simple.xml

```diff
@@ -3,10 +3,16 @@
   <nuclide name="I135" decay_modes="1" reactions="1" half_life="2.36520E+04">
     <decay type="beta" target="Xe135" branching_ratio="1.0" />
     <reaction type="(n,gamma)" Q="0.0" target="Xe136" /> <!-- Not precisely true, but whatever -->
+    <source type="discrete" particle="photon">
```

Comment:
I guess if we have type set to discrete there are also other options here. Perhaps we could include some of the other options in this chain file